src/design-tokens/tier-1-definitions/typography-presets.css

```diff
@@ -106,9 +106,9 @@
 @define-mixin eds-typography-preset-8 $important {
   font-family: var(--eds-font-family-tertiary) $important;
   font-weight: var(--eds-font-weight-regular) $important;
-  font-size: var(--eds-font-size-md) $important;
+  font-size: var(--eds-font-size-md-2) $important;
   line-height: var(--eds-line-height-sm) $important;
-  letter-spacing: normal $important;
+  letter-spacing: 1.5px $important;
```

Comment:
`preset-8` is equal to the old `preset-9`. The kicker typography is now larger than the button typography, so it gets 8 and 9 is now the large button styles.